compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -67,6 +72,11 @@ final class Conversions(
       .addAllNodes(nodes.asJava)
       .addAllScenarioSteps(steps.asJava)
       .setLedgerTime(ledger.currentTime.micros)
+      .addAllActiveContracts(
+        ledger.ledgerData.activeContracts
+          .map[String](coid => coidToEventId(coid).toLedgerString)
```

Comment:
i think the view is still more efficient so you don’t build up the intermediate data structure.